src/gluonnlp/data/transforms.py

```diff
@@ -17,20 +17,22 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# pylint: disable=invalid-encoded-data
+# pylint: disable=invalid-encoded-data, too-many-lines
```

Comment:
How about splitting up this file as it seems to be getting too large? Can also be addressed in a separate PR if you feel uncomfortable doing it here & now

Reply:
What about moving XXTokenizers to tokenizers.py, and leave the remaining ones in transforms.py? I'm also ok with doing that in a separate PR.

Reply:
Yes, separate PR is also fine. Besides, haibins suggestion, alternatively creating a folder 'transforms' with an init file and splitting the current file up should be possible